03-when-vector-data-dont-line-up-CRS.Rmd

```diff
@@ -17,7 +17,7 @@ vector object in R. It will also explore differences in units associated with
 different projections and how to reproject data using spTransform in R. Spatial
 data need to be in the same projection in order to successfully map and process
 them in non-gui tools such as R."
-code1: 02-csv-vector-raster-plotting.R
+code1: 03-when-vector-data-dont-line-up-CRS.R
```

Comment:
awesome. @mjones01 did you test these downloads (download the R file at the bottom of each page)? did they work? i found a few the other day that weren't working (and fixed them) in another repo.